docs/SNAPTEL.md

```diff
@@ -103,7 +111,7 @@ Example Usage
 
 In one terminal window, run snapteld (log level is set to 1 and signing is turned off for this example):
 ```
-$ $SNAP_PATH/bin/snapteld -l 1 -t 0
+$ snapteld -l 1 -o '' -t 0
```

Comment:
Yeah, because our default package configuration is `log-path: /var/log/snap/snapteld.log` (which is sensible), we must override `log-path` to '' to see the output on the cli. Otherwise snap will read the default config and write logs to `/var/log` and users mistakenly think nothing is happening on the system.